### DISCUSSION TITLE
What is defined as a "slow torrent"

### DISCUSSION THREAD

**Discussion by Treverr** [Drapersniper/Qbitrr#18] [Q&A] [answered]
I see there is an option and sometimes my torrents hit the "slow torrent" so it deletes it.

If im reading the code right 

`        elif (
            torrent.state_enum != TorrentStates.PAUSED_DOWNLOAD
            and torrent.state_enum.is_downloading
            and self.recently_queue.get(torrent.hash, torrent.added_on)
            < time_now - self.ignore_torrents_younger_than
            and 0 < maximum_eta < torrent.eta
            and not self.do_not_remove_slow
        ):`

It only deletes it if its downloading, and is greater than the young and the eta is greater than the max eta and this is enabled, is that all correct?

Does this functionally turn of removing torrents that will exceed the max time configured?

**Comment by Drapersniper** [comment, marked as answer]
Slow torrent is a torrent which the completion eta is longer than the specified maximum eta so you are right.

If the do not remove slow is enabled then it will not remove torrents even if their eta is greater than the maximum eta
